debify.gemspec

```diff
@@ -19,19 +19,18 @@ Gem::Specification.new do |spec|
   
   spec.add_dependency "gli"
   spec.add_dependency "docker-api", "~> 2.0"
-  spec.add_dependency "conjur-cli" , "~> 6"
-  spec.add_dependency "conjur-api", "~> 5"
-
+  #spec.add_dependency "conjur-cli" , "~> 6"
+  spec.add_dependency "conjur-api", "~> 5.pre", ">= 5.3.6"
```

Comment:
Should be changed to released version

Reply:
Released `conjur-api` and `conjur-cli`
Updated gemspec